### DISCUSSION TITLE
New payment method: barzahlen(.de)

### DISCUSSION THREAD

**Discussion by Lem** [pretix/pretix#1940] [Ideas]
I want to suggest barzahlen.de as a payment method.

Last time I've checked the API one was required to submit everything known about a customer. Now `show_stores_near:​address:*` seems to be optional.

The biggest point of it: You can pay with a code during grocery-shopping at supported stores. You could use still your card or other electronic cash in the store but the ability to use cash would be great.
Only mail-address or phone-number of customer required.

As I'm not familiar with python I'm not able to provide a plugin for pretix.

API: https://docs.barzahlen.de/api/v2/#introduction

**Comment by raphaelm** [comment]
Sure, would be great to have a plugin for that.

**Comment by TimmThaler** [comment]
While searching for some inspirations to implement 'barzahlen' for pretix, I found some sources for ideas.

* https://github.com/Barzahlen/Barzahlen-Java
* https://github.com/Barzahlen/Barzahlen-PHP
* https://github.com/FlorianSteenbuck/python-barzahlen

I'd like to code something, but it might take some time since I'm new to python and django. 😁
Perhaps somebody likes to help in case I get stuck.
